src/garage/experiment/local_runner.py

```diff
@@ -54,13 +55,30 @@ def __init__(self, snapshot_config, max_cpus=1):
                                         snapshot_config.snapshot_gap)
 
         if max_cpus > 1:
+            # pylint: disable=import-outside-toplevel
             from garage.sampler import singleton_pool
             singleton_pool.initialize(max_cpus)
-        self.has_setup = False
-        self.plot = False
+        self._has_setup = False
+        self._plot = False
 
         self._setup_args = None
-        self.train_args = None
+        self._train_args = None
+        self._stats = types.SimpleNamespace(total_itr=0,
+                                            total_env_steps=0,
+                                            total_epoch=0,
+                                            last_path=None)
+
+        self.algo = None
+        self.env = None
+        self.policy = None
```

Comment:
can you make these private? if they need to be part of the API, you can add properties for them.